### PR TITLE
Added ability to pass parameters along with the command via the External Program Plugin

### PR DESCRIPTION
**Pull request by bigmutty**
This is a fix to Issue #697.

### CODE REVIEW

File: src/JuliusSweetland.OptiKey.StandardPlugins/Native/ExternalProgram.cs [hunk 1]
```diff
@@ -17,9 +18,12 @@ namespace JuliusSweetland.OptiKey.StandardPlugins
     public class ExternalProgram
     {
         // Simply run it.
-        public void RUN(string command)
+        public void RUN(string command, string parameters = "")
```
Comment (JuliusSweetland) [inline src/JuliusSweetland.OptiKey.StandardPlugins/Native/ExternalProgram.cs:21]: Could we change to:
, string parameters = null

File: src/JuliusSweetland.OptiKey.StandardPlugins/Native/ExternalProgram.cs [hunk 2]
```diff
@@ -17,9 +18,12 @@ namespace JuliusSweetland.OptiKey.StandardPlugins
...
         {
-            Process.Start(command);
+            if (parameters == "")
```
Comment (JuliusSweetland) [inline src/JuliusSweetland.OptiKey.StandardPlugins/Native/ExternalProgram.cs:23]: Could we change to:
if (string.IsNullOrEmpty(parameters))

**Comment by JuliusSweetland** [comment]
@bigmutty Hi Daryl. Thanks for this - could we might 2 minor tweaks and then it's good to go.

**Comment by bigmutty** [comment]
Ok, I made those changes. Thanks for catching them.

**Comment by JuliusSweetland** [comment]
@bigmutty thanks. Could you also send over your dynamic keyboards to recreate #691 and I'll look at both tomorrow evening? Thx

**Pull request merged**